meetings/2022-05-09.md

```diff
@@ -0,0 +1,61 @@
+# W3C Solid Community Group: Test Suite Panel
+
+* Date: 2022-05-09T14:00:00Z
+* Call: https://meet.jit.si/solid-test-suite
+* Chat: https://gitter.im/solid/test-suite
+* Repository: https://github.com/test-suite-panel
+
+
+## Present
+* [Sarven Capadisli](https://csarven.ca/#i)
+
+---
+
+## Announcements
+
+### Meeting Recordings and Transcripts
+* No audio or video recording, or automated transcripts without consent. Meetings are transcribed and made public. If consent is withheld by anyone, recording/retention must not occur.
+* Join queue to talk.
+
+
+### Participation and Code of Conduct
+* [Join the W3C Solid Community Group](https://www.w3.org/community/solid/join), [W3C Account Request](http://www.w3.org/accounts/request), [W3C Community Contributor License Agreement](https://www.w3.org/community/about/agreements/cla/)
+* [Solid Code of Conduct](https://github.com/solid/process/blob/main/code-of-conduct.md), [Positive Work Environment at W3C: Code of Ethics and Professional Conduct](https://www.w3.org/Consortium/cepc/)
+* Operating principle for effective participation is to allow access across disabilities, across country borders, and across time. Feedback on tooling and meeting timing is welcome.
+* If this is your first time, welcome! please introduce yourself.
+
+
+### Scribes
+* name
+
+
+### Introductions
+* name: text
+
+---
+
+## Topics
+
+* SC: The Test Suite Panel is a "horizontal" panel. We need to revisit/clarify our understanding of the Test Suite Panel alongside other panels in the CG as well as contributors to specs and implementers.
+
+
+### Test Suite Panel Charter
+* SC: We need to stabilise a mutual understanding and a coordination plan. To advance our work, we need to agree on a charter that details its goals, scope, deliverables, decision policy (e.g., along the lines of https://github.com/solid/process/blob/main/notifications-panel-charter.md ).
+* SC: Here are some proposed actions. In these proposals, it is assumed that:
+  * Anyone can write a test.
+  * Anyone can contribute to the actions.
+  * Tests and reports that are not approved for publication at locations acknowledged by the Editors Team is out of scope for the Charter.
```

Comment:
So far our reports have not been approved by them at all. So unless they show signs of changing their stance, that would basically mean signing a resignation letter, which we're obviously not about to do.

Reply:
What I'm saying is, let's discuss this in the meeting, not take it as an assumption.

Reply:
The mutual understanding is for PROPOSALs 1-3. If a report ticks all the boxes as authoritative, yes, it is still possible for the software maintainer to decide not have the reports published. The likely case is that the software maintainer will want to publicise their project's conformance to specifications.

But yes, we can review of all this in the meeting.